great_expectations/rule_based_profiler/data_assistant/onboarding_data_assistant.py

```diff
@@ -291,18 +291,12 @@ def _build_numeric_columns_low_precision_rule() -> Rule:
             bucketize_data=True,
             json_serialize=True,
         )
-        column_column_quantile_values_metric_multi_batch_parameter_builder_for_metrics: ParameterBuilder = DataAssistant.commonly_used_parameter_builders.get_column_quantile_values_metric_multi_batch_parameter_builder(
-            json_serialize=True
-        )
         column_min_metric_multi_batch_parameter_builder_for_metrics: ParameterBuilder = DataAssistant.commonly_used_parameter_builders.get_column_min_metric_multi_batch_parameter_builder(
             json_serialize=True
         )
         column_max_metric_multi_batch_parameter_builder_for_metrics: ParameterBuilder = DataAssistant.commonly_used_parameter_builders.get_column_max_metric_multi_batch_parameter_builder(
             json_serialize=True
         )
-        column_median_metric_multi_batch_parameter_builder_for_metrics: ParameterBuilder = DataAssistant.commonly_used_parameter_builders.get_column_median_metric_multi_batch_parameter_builder(
```

Comment:
If min and max belong in this rule, why wouldn't median? What if all of your values are percentages with 2 decimal places?